watertap/examples/flowsheets/mvc/tests/test_mvc_single_stage.py

```diff
@@ -0,0 +1,341 @@
+        assert value(m.fs.compressor.efficiency) == 0.8
+
+        # 0 TDS in distillate
+        assert (
+            m.fs.tb_distillate.properties_out[0]
+            .flow_mass_phase_comp["Liq", "TDS"]
+            .is_fixed()
+        )
+        assert (
+            value(
+                m.fs.tb_distillate.properties_out[0].flow_mass_phase_comp["Liq", "TDS"]
+            )
+            == 1e-5
+        )
+
+        # Costing
+        assert m.fs.costing.factor_total_investment.is_fixed()
+        assert value(m.fs.costing.factor_total_investment) == 2
+        assert m.fs.costing.heat_exchanger.material_factor_cost.is_fixed()
+        assert value(m.fs.costing.heat_exchanger.material_factor_cost) == 5
+        assert m.fs.costing.evaporator.material_factor_cost.is_fixed()
+        assert value(m.fs.costing.evaporator.material_factor_cost) == 5
+        assert m.fs.costing.compressor.unit_cost.is_fixed()
+        assert value(m.fs.costing.compressor.unit_cost) == 7364
+
+        # Temperature upper bounds
+        assert value(m.fs.evaporator.properties_vapor[0].temperature.ub) == 348.15
+        assert (
+            value(m.fs.compressor.control_volume.properties_out[0].temperature.ub)
+            == 450
+        )
+
+        # check degrees of freedom
+        assert degrees_of_freedom(m) == 1
+
+    @pytest.mark.component
+    def test_initialize_system(self, mvc_single_stage):
+        m = mvc_single_stage
+
+        initialize_system(m)
+
+        # mass flows in evaporator
+        assert value(
+            m.fs.evaporator.properties_vapor[0].flow_mass_phase_comp["Vap", "H2O"]
+        ) == pytest.approx(22.222, rel=1e-3)
+        # assert value(
+        #     m.fs.evaporator.properties_vapor[0].flow_mass_phase_comp["Liq", "H2O"]
+        # ) == pytest.approx(1e-8, rel=1e-3)
+        assert value(
+            m.fs.evaporator.properties_brine[0].flow_mass_phase_comp["Liq", "H2O"]
+        ) == pytest.approx(17.777, rel=1e-3)
+        assert value(
+            m.fs.evaporator.properties_brine[0].flow_mass_phase_comp["Liq", "TDS"]
+        ) == pytest.approx(4.444, rel=1e-3)
+
+        # evaporator pressure
+        assert value(m.fs.evaporator.properties_vapor[0].pressure) == pytest.approx(
+            26.231e3, rel=1e-3
+        )
+
+        # compressor pressure outlet
+        assert value(m.fs.compressor.control_volume.properties_out[0].pressure) / value(
+            m.fs.compressor.control_volume.properties_in[0].pressure
+        ) == pytest.approx(1.6, rel=1e-3)
+
+        # external Q
+        assert value(m.fs.Q_ext[0]) == pytest.approx(0, 1e-3)
+
+    @pytest.mark.component
+    def test_simulation_Q_ext(self, mvc_single_stage):
+        m = mvc_single_stage
+
+        scale_costs(m)
+        fix_outlet_pressures(m)
+
+        assert degrees_of_freedom(m) == 1
+
+        m.fs.objective = Objective(expr=m.fs.Q_ext[0])
+        solver = get_solver()
+        results = solve(m, solver=solver)
```

Comment:
You should add an `assert_optimal_termination(results)` after this line, and wherever else you have a solve. This might show up in the tests as the reason for the failures on Linux (i.e., not that there are different solutions between Windows and Linux but rather that on Windows, you had optimal termination, while on Linux, you did not).